rollback_on_fail.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+rollback () {
+    wget https://github.com/Jellyfish-AI/jf_agent/archive/refs/tags/stable.tar.gz
+    tar -zxvf stable.tar.gz --directory ./
+    python jf_agent-stable/jf_agent/main.py
+}
+
+timeout --preserve-status 10h python jf_agent/main.py
+EXITCODE=$?
+# timeout will kill 143, config exception sys.exit's with 1, 
+# exceptions throw various; only a completed run exits with 0
+if [[ $EXITCODE -ne 0 ]]; then rollback; fi
```

Comment:
```suggestion
timeout --preserve-status 10h python jf_agent/main.py || rollback
```

Don't bother with exitcode parsing, `||` will run on anything non-zero (similarly, `&&` will run on `$? == 0`)